anonymizer/OpenDiffix.Service/JsonEncodersDecoders.fs

```diff
@@ -23,6 +23,16 @@ type CountInput =
   | Rows
   | Entities
 
+type RequestAnonParams =
+  {
+    Suppression: CommonTypes.SuppressionParams
```

Comment:
Why the `CommonTypes` prefix? The module is auto opened. Unless there is another type with such name?

Reply:
ah leftovers from time where I introduced a `SuppressionParams` here